docs/howto/event-stream.md

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 14
+title: Streaming events
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Streaming events from Vega
+
+:::danger Broken links
+* [schema documentation](https://docs.fairground.vega.xyz/api/graphql/data-node/subscription.doc.html#L92)
```

Comment:
https://docs.vega.xyz/docs/testnet/graphql/subscriptions/bus-events is the replacement page for this